go/signedexchange/cmd/dump-signedexchange/main.go

```diff
@@ -32,10 +36,43 @@ func run() error {
 
 	if *flagSignature {
 		fmt.Println(e.SignatureHeaderValue)
-	} else {
-		e.PrettyPrint(os.Stdout)
+		return nil
 	}
 
+	e.PrettyPrintHeaders(os.Stdout)
+
+	if *flagVerify {
+		if err := verify(e); err != nil {
+			return err
```

Comment:
`fmt.Println(err)`? Now the error is handled by `log.Fatal`, which I think is different from what we want.

Reply:
`verify` returns error not on verification errors, but when a bad command-line flag is given (e.g. -cert file doesn't exist). In such cases, probably we should exit with non-zero status.